src/app/resources/admin.py

```diff
@@ -83,3 +66,67 @@ def get(self, username):
             return UserSchema().dump(user_data), 200  # OK
         else:
             return 'User not found', 404  # Not Found
+
+
+@api.representation('/json')
+class AdminUserListDelete(Resource):
+    @api.expect(users_ids_list)
+    def delete(self):
```

Comment:
You can put this method in AdminUsersResource class which is located above (line 28). And after remove AdminUserListDelete class.

Reply:
I need new class to make new route url, according of our teble of requests. If I use AdminUserResource  the route will be /admin/users/ instead of /admin/users/list (in our table).